src/Transition/InputRulesFiles/Rules8-7-0-to-8-8-0.md

```diff
@@ -8,17 +8,22 @@ This was previously an Excel workbook that made for very difficult version contr
 
 Insert new blank field 5 (A5) (for System Inlet Air Node Name). 
 Shift all later fields down by one.
-=======
 
 # Object Change: `Table:TwoIndependentVariables`
 
 Insert new blank field 14 (A7) (for External File Name). 
 Shift all later fields down by one.
-=======
 
 # Object Change: `Output:Surfaces:List`
 
 The only change is for field F1, which is A1.  Logic to apply:
 
 if key = 'DecayCurvesfromZoneComponentLoads', change to = 'DecayCurvesFromComponentLoadsSummary'
 
+# Object Change: `UnitarySystemPerformance:Multispeed`
+
+Fields 1-4 remain the same.  
+After F4, insert one new blank field that represents the No Load Supply Air Flow Rate Ratio. 
+Shift all later fields down by one.
```

Comment:
Thanks for the reply.  I built it locally and I am getting a failure in the new unit test.  Says it failed to process the input.  I'm investigating.  If you could peek at the inputs in that unit test as well that would be great.  My first guess was there was a copy/paste and the new no-load field you added here wasn't in there or something, but it looks like it is there, so I'm not sure yet.